From 8574aedb51e6aaa49d84a5830495e8f76c4cf0e2 Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Fri, 19 Sep 2014 08:30:27 +0100
Subject: [PATCH 321/351] regulator: arizona-ldo1: Add additional supported
 voltage

Add support for the 1.175V mode on the LDO1 regulator for some Arizona
parts. This is needed as part of the sleep mode operation on WM8280.

Change-Id: I5fe1c7d482100a06babfa487587713d39ee16a53
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
---
 drivers/regulator/arizona-ldo1.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drivers/regulator/arizona-ldo1.c b/drivers/regulator/arizona-ldo1.c
index bd81a4c..9a6a3f7 100644
--- a/drivers/regulator/arizona-ldo1.c
+++ b/drivers/regulator/arizona-ldo1.c
@@ -167,7 +167,10 @@ static const struct regulator_init_data arizona_ldo1_dvfs = {
 
 static const struct regulator_init_data arizona_ldo1_default = {
 	.constraints = {
-		.valid_ops_mask = REGULATOR_CHANGE_STATUS,
+		.min_uV = 1175000,
+		.max_uV = 1200000,
+		.valid_ops_mask = REGULATOR_CHANGE_STATUS |
+				  REGULATOR_CHANGE_VOLTAGE,
 	},
 	.num_consumer_supplies = 1,
 };
-- 
1.7.9.5

